frontend/src/metabase/embedding/components/SharingMenu/test/QuestionSharingMenu.enterprise.unit.spec.tsx

```diff
@@ -3,31 +3,11 @@ import userEvent from "@testing-library/user-event";
 import { screen } from "__support__/ui";
 import { createMockCollection } from "metabase-types/api/mocks";
 
-import { openMenu, setupQuestionSharingMenu } from "./setup";
+import { setupQuestionSharingMenu } from "./setup";
 
 describe("QuestionSharingMenu > Enterprise", () => {
   describe("non-admins", () => {
-    it('Should show the "Create an alert" menu item to non-admins if the user has subscriptions/alerts permissions', async () => {
```

Comment:
These 2 tests were moved to new `QuestionMoreActionsMenu.enterprise.unit.spec.tsx`